be/src/olap/tablet_schema.h

```diff
@@ -280,6 +289,8 @@ class TabletColumn : public MetadataAdder<TabletColumn> {
     int32_t _variant_max_subcolumns_count = 0;
     PatternTypePB _pattern_type = PatternTypePB::MATCH_NAME_GLOB;
     bool _variant_enable_typed_paths_to_sparse = false;
+    // set variant_max_sparse_column_statistics_size
+    int32_t _variant_max_sparse_column_statistics_size = 10000;
```

Comment:
add a const value in const.h 
`static constexpr int DEFAULT_VARIANT_MAX_SPARSE_COLUMN_STATS_SIZR  = 10000`